src/utils/sortPeople.ts

```diff
@@ -0,0 +1,34 @@
+import { Person } from '../types';
+
+export function sortPeople(
+  sort: string | null,
+  order: string | null,
+  people: Person[],
+) {
+  const sortedPeople = people;
+
+  switch (sort) {
+    case 'sex':
+      sortedPeople.sort((person1, person2) =>
+        person1.sex.localeCompare(person2.sex),
+      );
+      break;
+    case 'name':
+      sortedPeople.sort((person1, person2) =>
+        person1.name.localeCompare(person2.name),
+      );
+      break;
+    case 'born':
+      sortedPeople.sort((person1, person2) => person1.born - person2.born);
+      break;
+    case 'died':
+      sortedPeople.sort((person1, person2) => person1.died - person2.died);
+      break;
+  }
+
+  if (order === 'desc') {
+    sortedPeople.sort(() => -1);
+  }
```

Comment:
The logic for sorting in descending order is incorrect. The current implementation sorts all elements to the same position, which does not reverse the array. Consider using `sortedPeople.reverse()` to reverse the order of the array after sorting it in ascending order.